config/ignition_edifice_ubuntu_focal.yaml

```diff
@@ -2,119 +2,126 @@ name: ignition_edifice_ubuntu_focal
-Package (% =libignition-sensors5-logical-camera) |\
-Package (% =libignition-sensors5-logical-camera-dev) |\
-Package (% =libignition-sensors5-magnetometer) |\
-Package (% =libignition-sensors5-magnetometer-dev) |\
-Package (% =libignition-sensors5-rendering) |\
-Package (% =libignition-sensors5-rendering-dev) |\
-Package (% =libignition-sensors5-rgbd-camera) |\
-Package (% =libignition-sensors5-rgbd-camera-dev) |\
-Package (% =libignition-sensors5-thermal-camera) |\
-Package (% =libignition-sensors5-thermal-camera-dev)), \
-$Version (% 5.1.0-*)) |\
-((Package (% =ignition-transport10) |\
-Package (% =libignition-transport10) |\
-Package (% libignition-transport10-core-dev) |\
-Package (% libignition-transport10-dbg) |\
-Package (% libignition-transport10-dev) |\
-Package (% libignition-transport10-log) |\
-Package (% libignition-transport10-log-dev)), \
-$Version (% 10.1.0-*)) |\
-((Package (% =ignition-utils1) |\
-Package (% libignition-utils1) |\
-Package (% libignition-utils1-core-dev) |\
-Package (% libignition-utils1-dbg) |\
-Package (% libignition-utils1-dev) |\
-Package (% libignition-utils1-cli) |\
-Package (% libignition-utils1-cli-dev)), \
-$Version (% 1.1.0-*)) |\
-((Package (% =sdformat11) |\
-Package (% =libsdformat11) |\
-Package (% =libsdformat11-dbg) |\
-Package (% =libsdformat11-dev) |\
-Package (% =sdformat11-doc) |\
-Package (% =sdformat11-sdf)), \
-$Version (% 11.3.0-*)) \
+(Package (= ignition-edifice)
+), $Version (% 1.0.3-1*) |\
+(Package (= ignition-common4) |\
+ Package (= libignition-common4) |\
+ Package (= libignition-common4-av) |\
+ Package (= libignition-common4-av-dev) |\
+ Package (= libignition-common4-core-dev) |\
+ Package (= libignition-common4-dev) |\
+ Package (= libignition-common4-events) |\
+ Package (= libignition-common4-events-dev) |\
+ Package (= libignition-common4-graphics) |\
+ Package (= libignition-common4-graphics-dev) |\
+ Package (= libignition-common4-profiler) |\
+ Package (= libignition-common4-profiler-dev)
+), $Version (% 4.5.0-1*) |\
+(Package (= ignition-fuel-tools6) |\
+ Package (= libignition-fuel-tools6) |\
+ Package (= libignition-fuel-tools6-dev)
+), $Version (% 6.2.0-1*) |\
+(Package (= ignition-gazebo5) |\
+ Package (= libignition-gazebo5) |\
+ Package (= libignition-gazebo5-dbg) |\
+ Package (= libignition-gazebo5-dev) |\
+ Package (= libignition-gazebo5-plugins)
+), $Version (% 5.4.0-1*) |\
+(Package (= ignition-gui5) |\
+ Package (= libignition-gui5) |\
+ Package (= libignition-gui5-dev)
+), $Version (% 5.5.0-1*) |\
+(Package (= ignition-launch4) |\
+ Package (= libignition-launch4) |\
+ Package (= libignition-launch4-dev)
+), $Version (% 4.1.0-1*) |\
+(Package (= ignition-math6) |\
+ Package (= libignition-math6) |\
+ Package (= libignition-math6-dbg) |\
+ Package (= libignition-math6-dev) |\
+ Package (= libignition-math6-eigen3-dev) |\
+ Package (= python3-ignition-math6) |\
+ Package (= ruby-ignition-math6)
+), $Version (% 6.10.0-1*) |\
```

Comment:
`ign-math6` is also on the Gazebo 11 / Citadel config, is it ok to have duplicates?

Reply:
I've probably swung my opinion on this from what it used to be.
When we were doing this by hand, it made sense (to me) to have each config cover disjoint sets of packages since otherwise you could end up in a situation where packages were out of "sync" between sets. (For example: the Citadel config could have math 6.9.0 while the Edifice config could have 6.10.0) but I think with the extra scripting to facilitate uploads it will make more sense to update everything at once and the chances for that mis-synchronization go down. So I think it is best to have every package needed by Edifice in the Edifice config and so on for the others.

Reply:
I updated the PR. Now https://github.com/ros-infrastructure/reprepro-updater/pull/146/commits/9b21a9f811266df3cf125e93a20d5bb0efa60233 _just_ bumps the versions and cmake and math are added in https://github.com/ros-infrastructure/reprepro-updater/pull/146/commits/5aa4818feedbd6c70dc0a6bc74b92edf3f06151f

Reply:
I'm ok with whatever works for you. I was just checking.